README.md

```diff
@@ -3,94 +3,229 @@ Wayne State University Senior Capstone Project to develop a semi-autonomous robo
 
 ### Setup Repository After SSH
-1. sudo apt-get update
-2. sudo apt-get upgrade
-3. sudo apt-get install bluez
-4. sudo apt-get install xboxdrv
-5. sudo raspi-config
-6. Select Interfacing Options
-7. Enable Camera
-8. Enable I2C
-9. Save changes
-10. git clone --recurse-submodules https://github.com/johnshamoon/robotic-leader-follower.git
-11. cd SunFounder_PiCar-V
-12. sudo ./install_dependencies
-13. picar servo-install
-    * Adjust wheel and camera servos during this step.
-14. export LD_LIBRARY_PATH=/usr/local/lib/ >> .bashrc
+1. Update package lists.
+> ```shell
+> sudo apt-get update
+> ```
+2. Update installed packages.
+> ```shell
+> sudo apt-get upgrade
+> ```
+3. Install the Bluez Bluetooth stack.
+> ```shell
+> sudo apt-get install bluez
+> ```
+4. Install the XBox gamepad driver.
+> ```shell
+> sudo apt-get install xboxdrv
+> ```
+5. Enter the Raspberry Pi configuration tool.
+> ```shell
+> sudo raspi-config
+> ```
+6. Select Interfacing Options.
+7. Enable Camera.
+8. Enable I2C.
+9. Save changes.
+10. Clone the repository and submodules.
+> ```console
+> git clone --recurse-submodules https://github.com/johnshamoon/robotic-leader-follower.git
+> ```
+11. Enter the SunFounder submodule.
+> ```shell
+> cd SunFounder_PiCar-V
+> ```
+12. Install the SunFounder dependencies.
+> ```shell
+> sudo ./install_dependencies
+> ```
+13. Adjust wheel and camera servos.
+> ```shell
+> picar servo-install
+> ```
+14. Add libraries path to bashrc.
+> ```shell
+> export LD_LIBRARY_PATH=/usr/local/lib/ >> ~/.bashrc
+> ```
 
 ### Connect Bluetooth Controller
-1. echo "sudo bash -c echo 1 > /sys/module/bluetooth/parameters/disable_ertm" >> .bashrc 
-2. sudo bluetoothctl
-3. power on
-4. agent on
-5. default-agent
-6. scan on
-7. pair $ADDRESS
-8. trust $ADDRESS
-9. connect $ADDRESS
-10. quit
-11. ls /dev/input/js0
-    * This should exist if the controller paired and connected properly.
+1. Disable ERTM on startup.
+> ```shell
+> echo "sudo su -s /bin/bash -c \"echo 1 > /sys/module/bluetooth/parameters/disable_ertm\"" >> .bashrc
+> ```
+2. Enter the Bluetooth command line control tool.
+> ```shell
+> sudo bluetoothctl
+> ```
+3. Power on the Bluetooth controller.
+> ```shell
+> power on
+> ```
+4. Enable the Bluetooth agent.
+> ```shell
+> agent on
+> ```
+5. Set the current agent as the default.
+> ```shell
+> default-agent
+> ```
+6. Scan for devices.
+> ```shell
+> scan on
+> ```
+7. Pair to the Bluetooth controller's Bluetooth address.
+> ```shell
+> pair $ADDRESS
+> ```
+8. Trust the Bluetooth controller.
+> ```shell
+> trust $ADDRESS
+> ```
+9. Connect to the Bluetooth controller.
+> ```shell
+> connect $ADDRESS
+> ```
+10. Exit bluetoothctl.
+> ```shell
+> quit
+> ```
+11. Check that the controller paired and connected properly.
+> ```shell
+> ls /dev/input/js0
+> ```
 
 ### Compiling and Installing OpenCV
 Installing OpenCV on Raspbian requires compiling it yourself. Here's are the steps to compile and link OpenCV to Python:
```

Comment:
'Installing OpenCV on Raspbian requires manual compilation. Here are the steps to compile and link OpenCV to Python:'